packages/compass-sidebar/src/components/multiple-connections/sidebar.tsx

```diff
@@ -37,6 +39,13 @@ import { useGlobalAppRegistry } from 'hadron-app-registry';
 
 const TOAST_TIMEOUT_MS = 5000; // 5 seconds.
 
+const SIDEBAR_MENU_ITEM_HEIGHT = spacing[1200];
+const SIDEBAR_HEADER_HEIGHT = spacing[1600];
+const COLLAPSED_LIST_HEIGHT = SIDEBAR_MENU_ITEM_HEIGHT;
+const MAX_EXPANDED_LIST_HEIGHT =
+  window.outerHeight - SIDEBAR_HEADER_HEIGHT - 2 * SIDEBAR_MENU_ITEM_HEIGHT;
+const DEFAULT_SAVED_CONNECTIONS_HEIGHT = 320;
```

Comment:
NIT: should this constant be defined inside "saved-connection-list" and used both here and in that component?